image/base/Dockerfile

```diff
@@ -127,9 +127,6 @@ RUN /tmp/install-redis
 ADD install-oxipng /tmp/install-oxipng
 RUN /tmp/install-oxipng
 
-RUN echo 'gem: --no-document' >> /usr/local/etc/gemrc &&\
```

Comment:
Note that the ruby image that this image is based off already does this: https://github.com/discourse/docker-ruby/blob/832c4cbc71ae9ad37b7206d83b73e473472a2e30/Dockerfile#L31-L34